utils/llm.py

```diff
@@ -120,7 +123,10 @@ def ask_llm(
         claude_adjust_factor += 0.2 * (actual_count / context_tokens)
         
     if cost_callback is not None:
-        cost_callback(litellm.completion_cost(response))
+        try:
+            cost_callback(litellm.completion_cost(response))
+        except:
+            cost_callback(0.001)
```

Comment:
What's the reason of this fallback cost?

Reply:
Same issue with the litellm. But maybe we dont need it and can just set the model to be local instead?

Reply:
Alright, understood. In the future we could actually prompt the user to add cost information and store it somewhere.